packages/shared/components/popups/TokenInformationPopup.svelte

```diff
@@ -4,7 +4,7 @@
     import { truncateString } from '@lib/helpers'
     import { openPopup, updatePopupProps } from '@lib/popup'
     import { AssetIcon, Button, Text, TextHint, AssetActionsButton, KeyValueBox } from 'shared/components'
-    import { FontWeightText } from '../Text.svelte'
+    import { FontWeight } from '../Text.svelte'
```

Comment:
Something for a future PR, but I think potentially have an auxiliary `ui` module with this sort of stuff. We have some functions in `lib/utils.ts` too specifically for UI like binding events and debouncing buttons - those could also probably go in there. Although it's in the docs, IMO just feels a bit strange importing an enum from a component.